docs/tutorials/toy-connector.md

```diff
@@ -265,23 +267,377 @@ def _call_api(endpoint, token):
 
 
 def check(config):
-    # Assert required configuration was provided
-    if "api_key" not in config or "stock_ticker" not in config:
```

Comment:
this check is superfluous because it is already that the object passed in meets the specified json schema for the config.